kuma/users/signal_handlers.py

```diff
@@ -21,10 +21,6 @@ def on_user_signed_up(sender, request, user, **kwargs):
     """
     Signal handler to be called when a given user has signed up.
     """
-    url = reverse('wiki.document', args=['MDN/Getting_started'])
-    msg = _('You have completed the first step of '
-            '<a href="%s">getting started with MDN</a>') % url
-    messages.success(request, msg)
```

Comment:
As you might have noticed, you'll have to run `make pythonlint` on this file.

Reply:
Yup indeed. Which extension do you use in VSCode to auto PEP8/Flake8 all the Python things?

Reply:
First of all, you need a virtualenv which has our version of `flake8` installed. Soon, you'll also need it so you get our version of `Black`. 
The virtualenv needs to be at least Python 3.7. You can try to run to run `pip install -r requirements/dev.txt` into this virtualenv. 

Once you have this virtualenv activated locally on the command like you should be able to run `make pythonlint` on the command line (outside of docker). 

Next you need to put the path to the `python` and stuff in the `.vscode/settings.json`. Here's mine:
```json
{
    "python.linting.pylintEnabled": false,
    "python.linting.flake8Enabled": true,
    "python.linting.enabled": true,
    "restructuredtext.confPath": "${workspaceFolder}/docs",
    "python.pythonPath": "/Users/peterbe/virtualenvs/kuma/bin/python"
}
```

I have no idea what that `restructuredtext.confPath` thing is. 

Also, in VSCode you need to select this virtualenv. Do you know how to do that? I'm no t actually sure if you need to once you've set up the `python.pythonPath` in the `.vscode/settings.json` file.